qa/sc_evm_test_debug_methods.py

```diff
@@ -156,6 +157,63 @@ def run_test(self):
+        }
+
+        # struct/opcode default tracer with verbosity boolean parameters
+        request = json.dumps({"jsonrpc": "2.0", "method": "debug_traceCall", "id": 40,
+                              "params": [trace_call_transaction, block_number,
+                                         {
+                                             "enableMemory": False,
+                                             "disableStack": True,
+                                             "disableStorage": True,
+                                             "enableReturnData": False
+                                         }]})
+        res = sc_node.ethv1(request)['result']
+        assert_true("error" not in res, "debug_traceCall failed for successful smart contract transaction")
+        trace_logs_length = len(res['structLogs'])
+        assert_true(trace_logs_length > 130, "unexpected number of trace logs, less than 130")
+
+        # call tracer - native tracer
+        request = json.dumps({"jsonrpc": "2.0", "method": "debug_traceCall", "id": 48, "params": [trace_call_transaction, block_number, {"tracer": "callTracer"}]})
+        res = sc_node.ethv1(request)['result']
```

Comment:
Please use the existing mechanism for making RPC calls:
```suggestion
        res = sc_node.rpc_debug_traceCall(trace_call_transaction, block_number, {"tracer": "callTracer"})
```

Reply:
Any function call on a node starting with `rpc_` will automatically translate into an RPC call.

Reply:
Call syntax changed to the one proposed for this and the others `debug_traceCall` method calls